### ISSUE TITLE
RTL8812au driver

### ISSUE THREAD

**Issue by Basfet** [maxnet/berryboot#389]
RTL8812au driver is in the Raspbian, OSMC, LibreElec. And it is not in Berryboot. Why can not compile it at Berryboot?
https://github.com/gnab/rtl8812au

**Comment by maxnet** [comment]
>RTL8812au driver is in the Raspbian

Since when?

**Comment by Basfet** [comment]
> Since when?

https://www.raspberrypi.org/forums/viewtopic.php?p=1140517#p1140517

**Comment by maxnet** [comment]
So there is no driver in Raspbian.
You can compile one yourself.

Same with us.

**Comment by Basfet** [comment]
> So there is no driver in Raspbian.

> [quote]MrEngman[/quote]
Yes, there is a driver for your kernel version if you are using Raspbian.

How I can compile this driver for Berryboot?
